projects/angular/src/data/datagrid/built-in/filters/datagrid-numeric-filter.ts

```diff
@@ -140,6 +151,14 @@ export class DatagridNumericFilter<T = any>
     return this.minPlaceholder || this.commonStrings.keys.minValue;
   }
 
+  get datagridNumericFilterFromValue() {
+    return this.datagridNumericFilterFrom || this.commonStrings.keys.numericFilterLabelFrom;
+  }
+
+  get datagridNumericFilterToValue() {
+    return this.datagridNumericFilterTo || this.commonStrings.keys.numericFilterLabelTo;
+  }
```

Comment:
```suggestion
  get minLabelValue() {
    return this.datagridNumericFilterTo || this.commonStrings.keys.numericFilterLabelTo;
  }
  
  get maxLabelValue() {
    return this.datagridNumericFilterFrom || this.commonStrings.keys.numericFilterLabelFrom;
  }
```

Name and order these consistently with `maxPlaceholderValue` and `minPlaceholderValue`.